### PR TITLE
Fixed UI bug

### PR DESCRIPTION
**Pull request by kafle-ashish**
The UI bug has been eliminated. Top bar was added.
screenshot_20180922-185903_1366x768
screenshot_20180922-185956_1366x768

### CODE REVIEW

**Comment by jaykay12** [comment]
@codedump00 I really appreciate your contribution.
I actually want the layout of the web view, See Issue: #2 to be in the present form only. I only want the mobile-view to be responsive without any modifications in the web-view.

As of now, if you have correctly linked the External CSS File, `../static/styles/styles.css` with the html template, then kindly do these modifications in the PR:
- Copy all the Internal CSS & Inline CSS in the External CSS file to resolve Issue: #4 
- Revert all the layout changes
- Push new commits to this PR only.

Issue #10 which you have already resolved to a large extent using media queries will then be dealt with.

File: Web-App/templates/index.html [hunk 1]
```diff
@@ -56,272 +52,259 @@
                       <span class="carousel-control-next-icon"></span>
                     </a>
                   </div>
-            </div>
-
-            <h1 class="display-4" style="margin-top:10px; font-size:40px; font-weight:bold" align="center">CricAI: Smart Outcome Prediction System</h1>
 
             <form method="post">
-                    <div class="row" style="margin-top:20px">
-                        <div class="col-md-4">
```
Comment (jaykay12) [inline Web-App/templates/index.html:65]: Actually I am using bootstrap framework to handle the designing, `col-md-4` & `col-md-6` etc. are the bootstrap classes to handle Grids using bootstrap. No need to create custom classes for them

**Comment by jaykay12** [comment]
Resolves #4 #8

**Comment by msaoudallah** [comment]
hey @codedump00  , there is an error in result page because you're including main.js at the end, i think you should remove it

**Pull request merged**